kong/clustering/rpc/manager.lua

```diff
@@ -634,6 +634,13 @@ function _M:connect(premature, node_id, host, path, cert, key)
   end
 
   ::err::
+  local worker_events = assert(kong.worker_events)
```

Comment:
could we wrap these code into a `do end` block?

Reply:
The function goes to the end right after. So what's the reason?

Reply:
i think its ok, so resolved.